src/main/Apis/Transactions/TransactionsApi.cs

```diff
@@ -11,8 +11,8 @@ internal TransactionsApi(PayStackApi api)
             _api = api;
         }
 
-        public TransactionInitializeResponse Initialize(string email, int amount, string reference = null, bool makeReferenceUnique = false)
-            => Initialize(new TransactionInitializeRequest { Reference = reference, Email = email, AmountInKobo = amount }, makeReferenceUnique);
+        public TransactionInitializeResponse Initialize(string email, int amount, string currency = "NGN", string reference = null, bool makeReferenceUnique = false)
```

Comment:
Cool!

I'll rather we put `currency` as the last parameter so it won't break codes that already see the third optional parameters as `Reference`, viz:

```C#
Initialize(string email, int amount, string reference = null, bool makeReferenceUnique = false, string currency = "NGN")
```

Thanks for fix, really appreciate!